docs/aerodromes/Perth.md

```diff
@@ -31,20 +31,25 @@
 | JULIM | 21 |
 | CONNI | 21 |
 | WAVES | 21 |
-| BEVLY | 21 | 
+| BEVLY | 21 (heavy), 24 (up to A330/B787) | 
```

Comment:
I think cleaner and more standardised to just go with 24, 21 if operationally required. "up to A330/B787" a bit ambiguous too, even if it is used IRL. If someone disagrees then feel free to ignore me

Reply:
Fixed to `24, 21 (if operationally required)`